package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "ocis-hello",
   "version": "0.0.0",
-  "description": "",
+  "description": "An example ocis extension for Phoenix",
```

Comment:
I have left that empty intentionally, this package will never be published anywhere.

Reply:
Ok, will delete it 👍